homeassistant/components/weather/openweathermap.py

```diff
@@ -132,23 +141,51 @@ def wind_bearing(self):
 
     @property
     def attribution(self):
-        """Return the attribution."""
-        return ATTRIBUTION
+        """
+        Return the attribution.
+
+        Passing different attributes to correctly display
+        the time in the frontend weather card.
+        """
+        if self._mode == 'daily':
+            result = ATTRIBUTION_DAILY
+        else:
+            result = ATTRIBUTION_HOURLY
+        return result
```

Comment:
This is a hack and must be removed. It's a frontend issue and must be addressed there.